src/main/java/callofcactus/menu/LoginScreen.java

```diff
@@ -0,0 +1,196 @@
+
+        passwordTextfield = new TextField("", UISkins.getTextfieldSkin());
+        passwordTextfield.setPasswordCharacter('*');
+        passwordTextfield.setPasswordMode(true);
+        passwordTextfield.setHeight(50);
+        passwordTextfield.setWidth(350);
+        passwordTextfield.setColor(Color.WHITE);
+        passwordTextfield.setTextFieldListener(passwordTextFieldListener);
+
+        loginButton = new TextButton("Login", UISkins.getButtonSkin());
+        loginButton.setHeight(50);
+        loginButton.setWidth(350);
+        loginButton.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                super.clicked(event, x, y);
+                checkValidLogin();
+            }
+        });
+
+        invalidPasswordLabel = new Label("Invalid Password", UISkins.getLabelSkin());
+        invalidPasswordLabel.setColor(Color.RED);
+        invalidPasswordLabel.setVisible(false);
+
+        stage.addActor(usernameLabel);
+        stage.addActor(passwordLabel);
+        stage.addActor(invalidPasswordLabel);
+
+        stage.addActor(usernameTextfield);
+        stage.addActor(passwordTextfield);
+
+        stage.addActor(loginButton);
+
+        Gdx.input.setInputProcessor(stage);
+    }
+
+    private void positionUI() {
+        usernameLabelPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 + 250);
+        passwordLabelPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 + 150);
+
+        passwordTextFieldPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 + 100);
+        usernameTextFieldPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 + 200);
+
+        loginButtonPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 - 50);
+
+        invalidLoginLabelPosition = new Vector2(Gdx.graphics.getWidth() / 2 - 50, Gdx.graphics.getHeight() / 2 - 200);
+
+        usernameTextfield.setPosition(usernameTextFieldPosition.x, usernameTextFieldPosition.y);
+        passwordTextfield.setPosition(passwordTextFieldPosition.x, passwordTextFieldPosition.y);
+
+        usernameLabel.setPosition(usernameLabelPosition.x, usernameLabelPosition.y);
+        passwordLabel.setPosition(passwordLabelPosition.x, passwordLabelPosition.y);
+
+        loginButton.setPosition(loginButtonPosition.x, loginButtonPosition.y);
+
+        invalidPasswordLabel.setPosition(invalidLoginLabelPosition.x, invalidLoginLabelPosition.y);
+
+    }
+
+    @Override
+    public void show() {
+
+    }
+
+    @Override
+    public void render(float v) {
+        Gdx.gl.glClearColor(1, 1, 1, 1);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+
+        //  GUI code
+        backgroundRenderer.render(spriteBatch);
+
+
+        stage.draw();
+        stage.act(v);
+    }
+
+    @Override
+    public void resize(int i, int i1) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Document empty method body](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1388792641/issues/source?bid=2979575&fileBranchId=2980831#l145)